config/crd/experimental/gateway.networking.k8s.io_gateways.yaml

```diff
@@ -500,13 +514,27 @@ spec:
             description: Status defines the current state of Gateway.
             properties:
               addresses:
-                description: Addresses lists the IP addresses that have actually been
-                  bound to the Gateway. These addresses may differ from the addresses
+                description: "Addresses lists the IP addresses that have actually
+                  been bound to the Gateway. These addresses may differ from the addresses
                   in the Spec, e.g. if the Gateway automatically assigns an address
-                  from a reserved pool.
+                  from a reserved pool. \n <gateway:validateIPAddress>"
```

Comment:
We may want to strip "<gateway:validateIPAddress>"?

Reply:
Good catch, removed this and a few other `<gateway:experimental>` tags that didn't need to be in CRDs.